### PR TITLE
sub3suite: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
A free, open source, cross platform Intelligence gathering tool.

Source: https://github.com/3nock/sub3suite
Compiling details: https://github.com/3nock/sub3suite/blob/main/COMPILING.md

### CODE REVIEW

**Comment by D3vil0p3r** [comment]
Not sure if `qtcreator` package is needed to be in `depends=`

File: packages/sub3suite/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+prepare(){
+  cd $pkgname
+
+  git submodule init
```
Comment (noraj) [inline packages/sub3suite/PKGBUILD:30]: I'm not sure of the way to package git submodules, let's wait @noptrix review

Comment (D3vil0p3r) [reply packages/sub3suite/PKGBUILD:30]: > I'm not sure of the way to package git submodules, let's wait @noptrix review

Ok, I tested it. It was the only cleanest and elegant way for submodules. If @noptrix approves, we could create a [PKGBUILD template](https://github.com/BlackArch/blackarch-pkgbuilds) for cases dealing with git submodules.

**Comment by noptrix** [comment]
thanks guys. yes, `prepare()` is correc tplace here ot handle it this way.

**Comment by noptrix** [comment]
@D3vil0p3r btw, when something gets compiled then `arch=('x86_64' 'aarch64')` rather than `arch=('any')`.

**Comment by D3vil0p3r** [comment]
Thank you @noptrix I will take a note about it.

**Comment by D3vil0p3r** [comment]
> thanks guys. yes, `prepare()` is correc tplace here ot handle it this way.

@noraj at this point could it be useful if we create a PKGBUILD template based on submodules for https://github.com/BlackArch/blackarch-pkgbuilds?

**Pull request merged**